content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -26,11 +26,62 @@ scheduling and evicting Pods.
 
 ## QoS classes
 
-When Kubernetes creates a Pod it assigns one of these QoS classes to the Pod:
+Quality of Services classes classify pods into one of three types: Guaranteed, Burstable, and BestEffort.
+The classification of pods is based on the [resource limits and requests](/docs/concepts/configuration/manage-resources-containers/).
+Each classification has effects on the scheduling policy and Eviction policy of a pod as well as on the resource usage of nodes.
+
+### Guaranteed
+
+To be sure a pod will be included in the Guaranteed QoS, every container in the pod must have
+resource(memory and CPU) limits and requests set, and they both have to be the same. The scheduler will make sure that
+a pod running in a Guaranteed QoS class is assigned to a node that has enough resources to fulfill its resource requests.
+A pod running in a Guaranteed QoS class can make use of exclusive CPUs  thanks to the [static](/docs/tasks/administer-cluster/cpu-management-policies/#static-policy) CPU management policy.
+
+When a node becomes [out of resource](/docs/tasks/administer-cluster/out-of-resource/), QoS classification helps kubelet
+to prioritize pods when the node needs to reclaim resources or when `oom_killer` needs to kill containers.
```

Comment:
```suggest
to prioritize Pods when the node needs to reclaim resources or when the operating system's out-of-memory handling is looking for processes to terminate.
```
?

The OOM killer might be set up to run with knowledge of containers and cgroups all the same, but that depends on the container runtime.

Reply:
it's intro for benefits of QoS classification I explain in more detail in the paragraph
```suggestion
to prioritize Pods when the kubelet receives `out-of-resource` signals or  when the operating system's out-of-memory handling is looking for processes to terminate.
```

Reply:
Only some container runtimes are concerned with `oom_killer` behavior (the Pod might be running on Windows, for example). I'd like the page to be broadly applicable.

Reply:
```suggestion
to prioritize Pods when the node needs to reclaim memory. The kubelet orders pods by their usage of memory,
```
I think there is no need to talk about `oom_killer` here this [`comment`](https://github.com/kubernetes/website/pull/20876/files#r439850968) is enough, do you agree?